pkg/controller/operators/olm/operator.go

```diff
@@ -413,6 +477,12 @@ func (a *Operator) syncClusterServiceVersion(obj interface{}) (syncError error)
 
 	operatorGroup := a.operatorGroupForActiveCSV(logger, updatedCSV)
 	if operatorGroup == nil {
+		logger.WithField("reason", "no operatorgroup found for active CSV").Info("skipping CSV resource copy to target namespaces")
+		return
+	}
+
+	if updatedCSV.Status.Phase == v1alpha1.CSVPhaseFailed && updatedCSV.Status.Reason == v1alpha1.CSVReasonInterOperatorGroupOwnerConflict {
+		logger.WithField("reason", updatedCSV.Status.Message).Info("skipping CSV resource copy to target namespaces")
 		return
```

Comment:
In the future we should enable status updates to copied CSVs in namespaces that aren't intersecting.